src/core/Api.js

```diff
@@ -102,4 +104,22 @@ const API = {
         instance.get(`conferences?_sort=start&_order=DESC&_start=${start}&_end=${end}`),
 };
 
+axios.interceptors.response.use((response) => {
+    // Do something with response data
+    return response;
+}, (error) => {
+    if (!error.response) {
+        // Axios error object. Contains it's own "message" in the error object.
+        return Promise.reject(error);
+    }
+
+    if (error.response.status === 440 || error.response.status === 401) {
+        store.dispatch(removeToken());
+        window.location.replace('/home');
```

Comment:
From what I've known, [the preferred way](https://stackoverflow.com/questions/42123261/programmatically-navigate-using-react-router-v4) to navigate somewhere in React Router is not this one.